lib/analytics.js

```diff
@@ -114,13 +126,18 @@ Analytics.prototype.init = Analytics.prototype.initialize = function(settings, o
     if (!Integration) delete settings[name];
   }, settings);
 
-  // add integrations
-  each(function(opts, name) {
-    var Integration = self.Integrations[name];
-    var integration = new Integration(clone(opts));
-    self.log('initialize %o - %o', name, opts);
-    self.add(integration);
-  }, settings);
+  if (doNotTrack) {
+    // Clear manually added integrations
+    this._integrations = {};
```

Comment:
> You wouldn't receive any data if this doNotTrack value was set to true. 

you would if the our customer's users have do not track in their browser enabled, but our customer hasn't enable this option.

There are 2 setttings.
One is by your as a Segment customer. You enable `doNotTrack` in your Segment workspace. The second is by our customer's users. They enable `doNotTrack` in their browser. If both are true, a.js won't send the event. Does that make sense?